docs/content/preview/explore/transactions/explicit-locking.md

```diff
@@ -34,20 +34,18 @@ The types of row locks currently supported are:
 
 ## Example
 
-The following example uses the `FOR UPDATE` row lock with the [Fail-on-Conflict](../../../architecture/transactions/concurrency-control/#fail-on-conflict) concurrency control policy. First, a row is selected for update, thereby locking it, and subsequently updated. A concurrent transaction should not be able to abort this transaction by updating the value of that row after the row is locked.
+The following example uses the `FOR UPDATE` row lock with the [fail-on-conflict](../../../architecture/transactions/concurrency-control/#fail-on-conflict) concurrency control policy. First, a row is selected for update, thereby locking it, and subsequently updated. A concurrent transaction should not be able to abort this transaction by updating the value of that row after the row is locked.
 
-{{% explore-setup-single %}}
-
-To try out this scenario, first create an example table with sample data, as follows:
+Before you can start using the example, you need to [set up a YugabyteDB universe](../../#set-up-yugabytedb-universe). When done, create an example table with sample data, as follows:
```

Comment:
Is the explore-setup-single shortcode removed because of the  cluster usage instead on universe? If thats the case , then the shortcode should  be modified instead. You can find it under layouts-> shortcodes- >explore-setup-single.html

Reply:
Actually, with the new version of https://docs.yugabyte.com/preview/explore/#set-up-yugabytedb-universe, the contents of the Setup box is not needed. All we need is to ask the user to create a universe and provide a link.

Reply:
I think we need to use this box because the "explore-setup-single" shortcode talks about creating a single node cluster locally or using YBM and not a multi-node cluster. So YBA won't be applicable for all these places where we have mentioned the single node cluster creation right?